From cc5433b3513477a3ccf6addac4b67fb4e2884d48 Mon Sep 17 00:00:00 2001
From: Satrajit Ghosh <satra@mit.edu>
Date: Mon, 22 Oct 2012 00:28:29 -0400
Subject: [PATCH] fix: github authentication

---
 tools/github.py |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tools/github.py b/tools/github.py
index 01e7fb6..619ace6 100644
--- a/tools/github.py
+++ b/tools/github.py
@@ -42,7 +42,12 @@ def create_hash_map():
     """
 
     hashmap = {}
+    from base64 import encodestring as base64
+    import pwd
+    login_name = pwd.getpwuid(os.geteuid())[0]
     conn = httplib.HTTPSConnection("api.github.com")
+    conn.request("GET", "/repos/nipy/nipype",
+                 headers={'Authorization': 'Basic %s' % base64(login_name)})
     try:
         conn.request("GET", "/repos/nipy/nipype/git/trees/master?recursive=1")
     except:
-- 
1.7.10.4

