From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF: install/ship tests/use_resources script

--- a/setup.py
+++ b/setup.py
@@ -436,7 +436,8 @@ def main(**extra_args):
                          pjoin('pipeline', 'engine', 'report_template.html'),
                          pjoin('external', 'd3.js'),
                          pjoin('interfaces', 'script_templates', '*'),
-                         pjoin('interfaces', 'tests', 'realign_json.json')
+                         pjoin('interfaces', 'tests', 'realign_json.json'),
+                         pjoin('interfaces', 'tests', 'use_resources'),
                          ]},
           scripts=glob('bin/*'),
           cmdclass=cmdclass,
