From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF: include nipype.workflows.data into setup.py

--- a/nipype/workflows/setup.py
+++ b/nipype/workflows/setup.py
@@ -5,6 +5,7 @@ def configuration(parent_package='',top_
 
     config = Configuration('workflows', parent_package, top_path)
 
+    config.add_subpackage('data')
     config.add_subpackage('dmri')
     config.add_subpackage('fmri')
     config.add_subpackage('graph')
